#! /usr/bin/env python

PACKAGE='ece6460_radar_example'

from dynamic_reconfigure.msg import SensorLevels
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       Name                Type       Lvl    Description                                      Default   Min     Max
gen.add('filter_objects',   bool_t,    0,    "Apply velocity filtering to incoming objects",   True)
gen.add('vel_thres',        double_t,  0,    "Global velocity thres for static objects",       2.0,      0.01,   10.0)

exit(gen.generate(PACKAGE, PACKAGE, "RadarExample"))
